src/main/java/dk/xam/starfish/Starfish.java

```diff
@@ -134,7 +146,7 @@ public static void editConfig()throws Exception{
             
             id=Integer.parseInt(reader.readLine());
             
-            if(id==1){ide="code";System.out.println("\n--------Selected IDE:VsCode--------");break;}
+            if(id==1){ide=isWindows()?"code.cmd":"code";System.out.println("\n--------Selected IDE:VsCode--------");break;}
```

Comment:
is this only relevant for code ? no diff for the others on windows ?

Reply:
- I haven't tested for eclipse and IDEA.
- I'll try doing that.